packages/flutter/test/material/app_test.dart

```diff
@@ -117,4 +117,16 @@ void main() {
     expect(buildCounter, 2);
   });
 
+  testWidgets('Cannot pop the initial route', (WidgetTester tester) async {
```

Comment:
I see you've added the isFirst check in two places.  Does this test cover both (assuming your goal was to test both)?